docs/pages/kubernetes-access/helm/guides/custom.mdx

```diff
@@ -0,0 +1,273 @@
+---
+title: Running a Teleport cluster with a custom configuration using Helm
+description: Install and configure a Teleport cluster with a custom configuration using Helm
+---
+
+In this guide, we'll go through how to set up a Teleport cluster in Kubernetes using a custom [`teleport.yaml`](../../../config-reference.mdx)
+config file using Teleport Helm charts.
+
+This setup can be useful when you already have an existing Teleport cluster and would like to start running it in Kubernetes, or when
+migrating your setup from an older version of the Helm charts.
+
+### Install Helm
+
+Teleport's charts require the use of Helm version 3. You can [install Helm 3 by following these instructions](https://helm.sh/docs/intro/install/).
+
+Throughout this guide, we will assume that you have the `helm` and `kubectl` binaries available in your `PATH`:
+
+```bash
+$ helm version
+version.BuildInfo{Version:"v3.4.2"}
+
+$ kubectl version
+Client Version: version.Info{Major:"1", Minor:"17+"}
+Server Version: version.Info{Major:"1", Minor:"17+"}
+```
+
+### Add the Teleport Helm chart repository
+
+To allow Helm to install charts which are hosted in the Teleport Helm repository, use `helm repo add`:
+
+```bash
+helm repo add teleport https://charts.releases.teleport.dev
+```
+
+Once installed, you can search repositories for charts using the `helm search repo` command:
+
+```bash
+helm search repo teleport
+```
+
+To update the cache of charts from the remote repository, run `helm repo update`:
+
+```bash
+helm repo update
+```
+
+### Setting up a Teleport cluster with Helm using a custom config
+
+In `custom` mode, the `teleport-cluster` Helm chart does not create a `ConfigMap` containing a `teleport.yaml` file for you, but
+expects that you will provide this yourself.
+
+For this example, we'll be using this `teleport.yaml` configuration file (with appropriately complex static tokens):
```

Comment:
Nit: at this point user does not know what tokens we are referring to yet.

Reply:
Good point, added a link to the static tokens part of the admin guide.